### ISSUE TITLE
TSA secret always required for TUF deployment

### ISSUE THREAD

**Issue by xoanmi** [sigstore/helm-charts#735]
**Description**

Currently, the TSA secret is consistently mandated for [TUF deployment](https://github.com/sigstore/helm-charts/blob/main/charts/tuf/templates/deployment.yaml#L66-L70), despite its optional nature. Could consideration be given to implementing a flag that allows for the exclusion of this secret setup if deemed unnecessary?"

**Comment by vipulagarwal** [comment]
There is some activity on #744 but if anyone is blocked by this can try the following:
```
  tsa:
    create: true
    name: tsa-cert-chain
    key: cert-chain
    path: dummyfile
    value: "dummyvalue"
```

This will make the `dummyfile` available in your tuf root though.

**Comment by vipulagarwal** [comment]
@bobcallaway can we close this issue? It has been fixed in #744
